docs/cli/dotnet-run.md

```diff
@@ -0,0 +1,25 @@
+dotnet-run
+==========
+
+## NAME 
+
+dotnet-run -- Runs source code 'in-place' without any explicit compile or launch commands.
+
+## SYNOPSIS
+
+`dotnet run [--verbose]`
```

Comment:
nearly all of the other synopsis are 'dotnet-verb'. why isn't this 'dotnet-run'?